recipe.cake

```diff
@@ -1,22 +1,23 @@
-#load nuget:?package=Cake.Recipe&version=1.0.0
+#load nuget:?package=Cake.Recipe&version=3.0.1
 
 Environment.SetVariableNames();
 
 BuildParameters.SetParameters(context: Context,
                             buildSystem: BuildSystem,
                             sourceDirectoryPath: "./src",
-                            title: "Cake.Grunt",
+                            title: "Cake.Gulp",
```

Comment:
@Tanzy are you sure, this change is correct? 
Renaming the `title` to `Cake.Gulp` will result in strange behaviors here, and maybe also in Cake.Gulp.

Reply:
good spot on this ta, changed